docs/reference/api_examples.yml

```diff
@@ -249,6 +249,11 @@
 
   doc: |
     Get a list of all datasets in a deployment.
+    This includes manifest (attached) and non-manifest (unattached) datasets.
+    Some examples of non-manifest datasets are datasets that are being created or
```

Comment:
Done....but it kind of implies that you can create non-manifest datasets....which perhaps we will allow in the future....I remember discussing a use-case with @exarkun which went something like "I want to prepare a batch of datasets in advance to avoid the delay of formatting them all when applications need to use them".